src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -1017,7 +1050,18 @@ private TexlNode ParseOperand()
 
                     if (AfterSpaceTokenId() == TokKind.ParenOpen)
                     {
+                        if (ident.Token.As<IdentToken>().Name.Value.ToString() == "Type")
+                        {
+                            ParseTrivia();
+                            TokEat(TokKind.ParenOpen);
+                            var node = ParseTypeLiteral();
+                            ParseTrivia();
+                            TokEat(TokKind.ParenClose);

```

Comment:
structurally, consider moving the tokEat, and parsetrivia calls into the ParseTypeLiteral function